include/proxy-wasm/context.h

```diff
@@ -142,8 +143,8 @@ class ContextBase : public RootInterface,
   ContextBase();                                                   // Testing.
   ContextBase(WasmBase *wasm);                                     // Vm Context.
   ContextBase(WasmBase *wasm, std::shared_ptr<PluginBase> plugin); // Root Context.
-  ContextBase(WasmBase *wasm, uint32_t parent_context_id,
-              std::shared_ptr<PluginBase> plugin); // Stream context.
+  ContextBase(WasmBase *wasm, uint32_t parent_context_id, std::shared_ptr<PluginBase> plugin,
+              std::shared_ptr<PluginHandleBase> plugin_handle); // Stream context.
```

Comment:
Should `PluginHandleBase` contain `std::shared_ptr<PluginBase>`, so that we wouldn't need to pass both?

Reply:
thanks, good point. will do

Reply:
done.